src/legacy/ui/public/vis/editors/default/components/agg_group.tsx

```diff
@@ -155,6 +164,7 @@ function DefaultEditorAggGroup({
                     isDraggable={stats.count > 1}
                     isLastBucket={groupName === AggGroupNames.Buckets && index === group.length - 1}
                     isRemovable={isAggRemovable(agg, group)}
+                    isDisabled={agg.schema.name === 'metric' ? isMetricAggregationDisabled : false}
```

Comment:
I see that `agg.schema.name === 'metric` is already checked in `getEnabledMetricAggsCount`, so there is no necessity to check it again.
```suggestion
                    isDisabled={isMetricAggregationDisabled}
```